docker-env/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM python:3.10-bullseye
+
+# Define working directory
+WORKDIR /source_code
+
+# Install requirements package for python with poetry
+RUN curl -sSL https://install.python-poetry.org | python3 -
+COPY pyproject.toml poetry.lock ./
+RUN /root/.local/bin/poetry install
+
+# Install cron
+RUN apt-get update
+RUN apt-get install -y cron
+
+# Install chrome in a specific version 109
+ARG CHROME_VERSION="109.0.5414.119-1"
```

Comment:
Je suppose que tu n'as pas pu faire autrement que d'installer chrome pour faire fonctionner le driver.
Le fait d'avoir une version fixée dans le dockerfile le lie aux dépendances python, il faudra que nous mettions à jour les 2 si nous décidons d'upgrade le scrapper python.
Ce n'est pas un soucis en soi pour le moment, cependant ce serait intéressant de voir comment on peut se passer de cette configuration hardcodée dans le futur.

Reply:
Salut Amine, Effectivement complètement d'accord avec toi sur le fait que pas très maintenable dans le temps.
J'ai essayé pas mal de chose mais j'ai vraiment été obligé d'installer la version Chrome 109 correspondante à la version utilisé par le ChromeDriver Python. Sinon j'avais l'erreur suivante : selenium.common.exceptions.WebDriverException: Message: unknown error: cannot connect to chrome at 127.0.0.1:56691
from session not created: This version of ChromeDriver only supports Chrome version 109
Current browser version is 108.0.5359.94
Donc j'ai décidé de faire comme ça mais effectivement il faut peut être prévoir une issue pour que la version de Chrome utilisé dans le fichier marine_traffic_scrapper.py s'adapte en fonction de celle installée dans l'environnement. A discuter mais effectivement ce point est surement à régler dans une autre issue :)